### ISSUE TITLE
Add real-time bandwidth usage feature

### ISSUE THREAD

**Issue by YoshiLoL0526** [henrypp/simplewall#1712]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

3.7.7

### Problem you are trying to solve

Currently, users of Simple Wall do not have real-time visibility into the bandwidth consumption of individual applications. This feature request aims to address the lack of insight into real-time bandwidth usage, which is crucial for managing network resources efficiently.

### Suggested solution

I propose adding two new columns within the "Applications" tab in the Simple Wall interface. These columns would display the real-time upload and download data for each application. By including these columns, users can quickly and easily monitor the upload and download bandwidth usage for each running application.

### Screenshots / Drawings / Technical details

If implemented, the "Applications" tab would show a list of applications with additional columns for "Upload" and "Download" data, displaying the real-time bandwidth usage for each application. Users would be able to readily observe the network activity of individual applications and take appropriate actions based on this information.

**Comment by Spheres-cu** [comment]
I think this pre-release fix this feat: [v.3.9.1](https://github.com/henrypp/simplewall/releases/tag/v.3.9.1)

**Comment by henrypp** [comment]
@Spheres-cu not exactly, before implement this feature (with new columns) i need to implement another feature #1643, so wait!
